### DISCUSSION TITLE
Create PR from mobile app

### DISCUSSION THREAD

**Discussion by jeebb** [community/community#3479] [Mobile] [answered]
I really want to create the PRs directly from the mobile app. Is it possible to do so?

**Comment by PranamBhat** [comment]
Hi @jeebb 

Check here : [https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests)

**Reply by jeebb** [reply]
Hi 

What I mean is "creating PR i n mobile app (e.g. GitHub app in iPad)", not using the web interface.

**Comment by PranamBhat** [comment, marked as answer]
Hi @jeebb 

This is wrong entry.

Kindly post your query over "Mobile Feedback" category : [https://github.com/github/feedback/discussions/categories/mobile-feedback](https://github.com/github/feedback/discussions/categories/mobile-feedback)

**Reply by duy-ng-12354** [reply]
Thank you. Let me divert it.

**Comment by PranamBhat** [comment]
@jeebb @duy-ng-12354 Kindly mark this as "**Answered**" and leave the discussion. Thank you.
